denizen_scripts/survival/repo-link/libs/shapes.dsc

```diff
@@ -212,7 +212,7 @@ cosmetic_command:
         - narrate "<&b>Activated cosmetic effect curve1"
         - flag player curve
       - while <player.has_flag[curve]||false>:
-        - if <player.is_spawned||false> == false:
+        - if !<player.is_spawned||false>:
```

Comment:
`EntityTag.is_spawned` is more for entities; players can be checked with just `playertag.is_online`
```suggestion
        - if !<player.is_online>:
```